pulse/pulse/ViewController.swift

```diff
@@ -7,12 +7,19 @@
 //
 
 import UIKit
+import Alamofire
+import SwiftyJSON
 
 class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
     }
-
+    
+    // Request to news api made here
+    func makeApiRequest(completionHandler: @escaping (Any) -> Void) {
+        var params: [String: String] = ["q": "ghana", "sortBy": "publishedAt"]
```

Comment:
Hardcoded for now to see how requests could be made. Obviously this will be more dynamic when we start building the app